### DISCUSSION TITLE
Not working on macOS Sequoia Version 15.2

### DISCUSSION THREAD

**Discussion by xRME-18** [cymian/mouseless#103] [General]
Left command work but after that when I select two key nothing happens I'm still interacting with the underlaying screen, the Mouseless overlay has no effect. only option to exit works.

**Comment by cymian** [comment]
Hi @xRME-18, thank you for the report.  Can you please supply the following information to help diagnose?

- version of mouseless
- any other apps running that might read or generate keyboard input, or automate windows?  (e.g. karabiner, alfred, etc.). a full list of apps with accessibility permissions and automation permissions would be most helpful
- can you please email me your .mouseless/logs/log.log to the email found at https://mouseless.click/support
- which keyboard layout(s) you use
- your config file

Hope we can figure it out soon!
